misc/python/materialize/checks/mzcompose_actions.py

```diff
@@ -62,10 +62,11 @@ def execute(self, e: Executor) -> None:
                 version_cargo == mz_version
             ), f"Materialize version mismatch, expected {version_cargo}, but got {mz_version}"
 
+        e.current_mz_version = mz_version
+
 
 class ConfigureMz(MzcomposeAction):
```

Comment:
I think we should be running the parts of ConfigureMz only once when the minimum version for that part is exceeded first. Alternatively those simple SETs should be much faster than they currently are.